src/p11_child/p11_child_common_utils.c

```diff
@@ -64,6 +64,38 @@ static struct cert_verify_opts *init_cert_verify_opts(TALLOC_CTX *mem_ctx)
 #define OCSP_DGST "ocsp_dgst="
 #define OCSP_DGST_LEN (sizeof(OCSP_DGST) -1)
 
+static errno_t parse_crl_files(char **opts, size_t c, struct cert_verify_opts *_opts)
```

Comment:
Hi,

maybe you can pass `&opts[c][CRL_FILE_LEN]` as first argument, then you can rename the first argument to `filename` or similar to avoid having two variables with more or less the same name?

bye,
Sumit

Reply:
Done